cypress_tests/cypress/integration/footers/upper_footers.spec.js

```diff
@@ -0,0 +1,33 @@
+describe("Upper footer tests", () => {
```

Comment:
this can be moved to `navbar.js` file too as it's also part of site navigation

Reply:
Addressed in subsequent commit